name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/SecuritiesChart.java

```diff
@@ -733,6 +736,7 @@ private void updateChart()
             ChartRange range = ChartRange.createFor(prices, chartInterval);
             if (range == null)
             {
+                chart.resetAxes();
                 chart.redraw();
```

Comment:
So, (perhaps) very last thing: Those calls to `redraw()` are redundant, because the method is once again called from the `finally` block at [line 855](https://github.com/buchen/portfolio/blob/master/name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/SecuritiesChart.java#L855). I don’t know if it really redraws twice, but if it does, it would be a waste of processing time. :) So just a `return` here (doesn’t even need the braces then), and setting the title before `return`ing above.